rest-api-0_1-swagger.yaml

```diff
@@ -0,0 +1,278 @@
+# Example YAML to get you started quickly.
+# Be aware that YAML has indentation based scoping.
+# Code completion support is available so start typing for available options.
```

Comment:
This is the API specification in swagger format that is also in the Slack chat. I thought it would be a good idea to keep it in the repo as well.